jetson/nav/rover.cpp

```diff
@@ -59,26 +59,15 @@ Target& Rover::RoverStatus::leftTarget()
     return mTargetLeft;
 } // leftTarget()
 
-Target& Rover::RoverStatus::rightTarget() 
-{
-    return mTargetRight;
-} // rightTarget()
-
-Target& Rover::RoverStatus::leftCacheTarget()
-{
-    return mCTargetLeft;
-} // leftCacheTarget()
-
-Target& Rover::RoverStatus::rightCacheTarget() 
-{
-    return mCTargetRight;
-} // rightCacheTarget()
-
-RadioSignalStrength& Rover::RoverStatus::radio() 
-{
+Target& Rover::RoverStatus::target2() {
```

Comment:
Need to remove this, target2 no longer exists

Reply:
will you remove it before the PR is merged in?